app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -82,33 +83,41 @@ The international relocation payment (IRP) is designed to cover some of the cost
 
 * other relocation expenses
 
-The IRP is a grant – you will not have to pay this money back. If you are eligible, you can only receive the IRP once.
+You can apply for the IRP between September 2024 and March 2025. If you are eligible, you will receive a maximum IRP of £10,000, in 2 instalments of £5,000 spread over 2 years.
+
+The IRP is available whether you teach full-time or part-time.
 
 [Learn more about teaching in England if you trained outside the UK](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
 
-## Criteria for teachers
+## Check your eligibility
+
+To be eligible for the maximum IRP of £10,000, you must meet all the eligibility requirements when you apply for both the first and second instalments.
+
+### Eligibility requirements for IRP instalment 1 (£5,000)
 
-To be eligible for the international relocation payment (IRP), you must be a teacher who meets all 3 of the following requirements.
+#### Teaching subject and school
 
-If your teaching job starts after 1 September 2024, please keep checking this page for updates to the eligibility criteria, which we’ll publish in due course.
+You must be:
+ 
+ * employed by and working in a state secondary school in England
+ * on a contract of employment lasting at least 1 year
 
-### 1. Teaching subject for teachers
+Your contract of employment must begin between 1 March 2024 and 1 May 2025.
+State schools are schools funded by the UK government. Secondary schools teach children aged 11 to 16 or 11 to 18. Schools for pupils aged 16 to 18 only are not eligible. Schools for pupils aged 5 to 18 are eligible, but you must teach the secondary age range (11 to 16 or 11 to 18).
```

Comment:
Need a new paragraph here (blank line above?) to separate from contract of employement sentence

Reply:
I still think this is hard to read mashed together - wasn't sure if you disagree or had made a change?

![image](https://github.com/user-attachments/assets/0928efc9-49dd-46d6-a29f-6c7931ab92de)